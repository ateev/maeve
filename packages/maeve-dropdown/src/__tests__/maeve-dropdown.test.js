import React from 'react';
import { shallow } from 'enzyme';
import MaeveInput from '../maeve-input.js';

test('render maeve input', () => {
  const component = shallow(
    <MaeveInput />
  );
  expect(component).toMatchSnapshot();
});