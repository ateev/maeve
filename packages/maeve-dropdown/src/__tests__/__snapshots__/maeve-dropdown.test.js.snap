exports[`test render maeve input 1`] = `
<styled.div>
  <styled.ul />
</styled.div>
`;
